DirectProgramming/DPC++/ProjectTemplates/cmake-fpga/README.md

```diff
@@ -79,6 +79,11 @@ The following instructions assume you are in the project's root folder.
      make report
      ```
      Locate the FPGA optimization report, `report.html`, in the `fpga_report.prj/reports/` directory.
+   - Use the following commands to build and run the template for FPGA simulator:
+     ```
+     make fpga_sim

```

Comment:
Wow I did not even know about this sample.
We don't even test it :/
Paul is working on a similar but up to date sample, I'll ask for it to be replaced + moved to the FPGA part of the code samples.